feature/experimental/isis/ate_tests/lsp_updates_test/lsp_updates_test.go

```diff
@@ -76,7 +76,7 @@ func TestOverloadBit(t *testing.T) {
 	if err := check.Equal(overloads.State(), uint32(olVal+1)).AwaitFor(time.Second*10, ts.DUTClient); err != nil {
 		t.Error(err)
 	}
-	if err := check.Equal(setBit.State(), true).AwaitFor(time.Second, ts.DUTClient); err != nil {
+	if err := check.Equal(setBit.State(), true).AwaitFor(time.Second*15, ts.DUTClient); err != nil {
```

Comment:
same.

Reply:
one second is not sufficient, test will fail with deadline exceeded error